tests/contrib/operators/test_mssql_to_gcs_operator.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import sys
+import unittest
+
+from airflow.contrib.operators.mssql_to_gcs import \
+    MsSqlToGoogleCloudStorageOperator
+from tests.compat import mock
+
+PY3 = sys.version_info[0] == 3
```

Comment:
In master branch we already dropped support to Python 2. So no need to have this (and the related lines)

Reply:
Resolved with [797c4e1](https://github.com/apache/airflow/pull/5077/commits/797c4e11c0c4eab873ed8e3a4843646665d6b407).